code/game/machinery/excelsior/ex_turret.dm

```diff
@@ -2,6 +2,7 @@
 #define TURRET_SECONDARY_TARGET 1
 #define TURRET_NOT_TARGET 0
```

Comment:
You'll also want to comment out these defines, else they'll just dangle around elsewhere.

Reply:
Tis not needed and not hurting anything.